packages/apps/spaces/pages/api/auth/[...nextauth].ts

```diff
@@ -38,6 +38,23 @@ export const authOptions: AuthOptions = {
         token.accessToken = account.access_token;
         token.id = (profile as GoogleProfile)?.id;
         token.playerIdentityId = account.providerAccountId;
+        token.email = (profile as GoogleProfile)?.email;
+
+        // Check if the email is available in the profile
+        if (profile && profile.email) {
+          token.email = profile.email;
+        } else {
```

Comment:
Pleae check for azure-ad as provider. Otherwise we use it as fallback for google logins also